### PR TITLE
SP9 - add flake8 support

### PR DESCRIPTION
**Pull request by kalbermattenm**
I had to add an empty section inside `development.ini`, but I do not really get why (see https://github.com/Pylons/pyramid/issues/3130)

This PR is not ready to be merged, as I would like to run flake8 on all Python files...

### CODE REVIEW

**Comment by kalbermattenm** [comment]
Ready... @voisardf, please review

File: sitn_portal/models.py [hunk 1]
```diff
@@ -1,16 +1,13 @@
 from sqlalchemy import (
     Column,
     Integer,
-    Text,
-    String,
 )
```
Comment (voisardf) [inline sitn_portal/models.py]: What's the reason you deleted both string options?

Comment (kalbermattenm) [inline sitn_portal/models.py]: There are not used in the "commented" model... But feel free to re-add them when you feel a need for it....

**Comment by kalbermattenm** [comment]
Thanks for the review, merging !

**Pull request merged**